kubernetes/windows/setup.ps1

```diff
@@ -20,6 +20,9 @@ Write-Host ('Creating folder structure')
     New-Item -Type Directory -Path /etc/config/settings/
     New-Item -Type Directory -Path /etc/config/adx/
 
+    New-Item -Type Directory -Path /opt/amalogswindows/state/
+    New-Item -Type Directory -Path /opt/amalogswindows/state/ContainerInventory/
```

Comment:
what will be the files under /opt/amalogswindows/state/?  Does it contains any files with *.dll,*.exe,*.so,*.ps1

Reply:
No it doesnt contain any of the files you mentioned